ansible/playbooks/start-redpanda.yml

```diff
@@ -61,3 +61,32 @@
         --api-urls {{ groups["redpanda"] | map('extract', hostvars) | map(attribute='private_ip') | product([':9644']) | map('join') | join(',') }}
       {% endif %}
     when: rack is defined and rack != -1
+
+  - name: enable shadow indexing
+    notify:
+      - restart redpanda-tuner
+      - restart redpanda
+    vars:
+      use_public_ips: "{{ advertise_public_ips | d() | bool }}"
+      this_node: "{{ inventory_hostname if use_public_ips else private_ip }}"
+    shell: |
+      sudo -u redpanda rpk cluster config set cloud_storage_bucket {{ si_bucket_name }} --api-urls {{ "%s:%d" % (this_node, 9644) }}
+      sudo -u redpanda rpk cluster config set cloud_storage_region {{ aws_region }} --api-urls {{ "%s:%d" % (this_node, 9644) }}
+      sudo -u redpanda rpk cluster config set cloud_storage_access_key ABCDEFGHIJKLMNOP --api-urls {{ "%s:%d" % (this_node, 9644) }}
+      sudo -u redpanda rpk cluster config set cloud_storage_secret_key 1234567890abcdefghijklmnop --api-urls {{ "%s:%d" % (this_node, 9644) }}
```

Comment:
Shouldn't this `access_key` and `secret_key` be overridden by an ansible var?

Reply:
The only reason these two values are here is because they are required by Redpanda, but they are never being used in this implementation. This instead relies on the permissions available from the associated IAM profile.

But I agree that we would need to replace these values with user-provided ones if that approach was being implemented. If we do decide to implement that approach, then I was hoping for that to be in a separate PR so that this one just focuses on the IAM profile approach.

Reply:
I think we should set them to something like "THISVALUENOTUSED" and a comment nearby to explain why.